### ISSUE TITLE
Enable knative-serving test

### ISSUE THREAD

**Issue by aerfio** [kyma-project/kyma#5938]
**Description**

As we agreed in https://github.com/kyma-project/community/issues/369 we are turning off flaky tests in [this](https://github.com/kyma-project/kyma/pull//5937) PR, and `knative-serving` is one of them. This needs to be re-enabled.

**Comment by nachtmaar** [comment]
Works on minikube on yesterday's master, might only occur in stability-checker. Taking a closer look now.

**Comment by aerfio** [comment]
**Update after quality SIG meeting:**

During the meeting it was discussed that disabling whole test suite is too much most of the time, as only few tests inside are really unstable. It was proposed that a person on duty on CI-Force should have an easy way to disable one particular test by regexp or some other mechanism.

**Comment by nachtmaar** [comment]
TL;DR

```
 textPayload: "	knative_serving_test.go:81: Cannot create test Service: services.serving.knative.dev "test-service" already exists
```
Possible caused by force deleting the knative service: https://github.com/kyma-project/kyma/blob/d2afb5838f900765287100a1b533527e2b4fb686/tests/knative-serving/knative_serving_test.go#L179

The two pictures show that the failure message always comes then the test failed:
<img width="799" alt="Screenshot 2019-10-10 at 18 05 22" src="https://user-images.githubusercontent.com/8013145/66587064-009fd600-eb8a-11e9-88f8-6c09a15d7aa6.png">
<img width="876" alt="Screenshot 2019-10-10 at 18 15 34" src="https://user-images.githubusercontent.com/8013145/66587065-01d10300-eb8a-11e9-805a-810eb225fd2b.png">

**Comment by nachtmaar** [comment]
Implemented ...
